docs/static/ds6/docs.min.js

```diff
@@ -73,18 +73,18 @@ $_mod.def("/makeup-floating-label$0.0.5/index",function(t,e,i,l,n){"use strict";
+$_mod.installed("@ebay/skin$7.4.1","makeup-active-descendant","0.2.0");
+$_mod.main("/makeup-active-descendant$0.2.0","");
+$_mod.installed("makeup-active-descendant$0.2.0","custom-event-polyfill","1.0.7");
```

Comment:
You might need to run `yarn` again then `yarn build` . These JS files shouldn't be changing.